### DISCUSSION TITLE
Add intrinsic fields to TraceToMetrics

### DISCUSSION THREAD

**Discussion by faustodavid** [grafana/grafana#65336] [Data Sources]
Grafana allows us to link traces to metrics by mapping span tags to a predefined metric query.
https://grafana.com/docs/grafana/latest/datasources/tempo/#provision-the-data-source

The feature works great but misses the intrinsic fields, which are default dimensions when using the Tempo Metrics Generator.

I suggest adding an extra check to the expression to look for `$__name`, `$__status`, and `$__kind`. 

https://github.com/grafana/grafana/blob/a423c7f22ebbc491325ab15c3434e3b95dc165d0/public/app/features/explore/TraceView/createSpanLink.tsx#L372

**Comment by ifrost** [comment]
@grafana/observability-traces-and-profiling

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82734. Feel free to continue the discussion around this there. Thank you!
